README.md

```diff
@@ -1,57 +1,33 @@
-# Project Name
-
-(short, 1-3 sentenced, description of the project)
-
-## Features
-
-This project framework provides the following features:
-
-* Feature 1
-* Feature 2
-* ...
-
-## Getting Started
-
-### Prerequisites
-
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
-
-### Installation
-
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
-
-### Quickstart
-(Add steps to get up and running quickly)
-
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
-
-
-## Demo
-
-A demo app is included to show how to use the project.
-
-To run the demo, follow these steps:
-
-(Add steps to start up the demo)
-
-1.
-2.
-3.
-
-## Resources
-
-(Any additional resources or related projects)
-
-- Link to supporting information
-- Link to similar sample
-- ...
+# Azure Communication Network Traversal sample
+
+### Prerequisites
+
+You need Node.js

```

Comment:
Other hero samples prerequisite sections look like this:
![image](https://user-images.githubusercontent.com/58831720/143081709-2c5e0588-f1d7-4924-93b0-2b4af823002e.png)
So instead of saying "You need Node.js", just have "Node.js" and the version and link to the download page.  
"Computer with camera" maybe instead "Computer with a camera"
Connection string isn't really a proper noun, "you will need the connection string" is probably better.